tests/test_offchain_jws.py

```diff
@@ -34,10 +34,28 @@ def test_deserialize_error_if_not_3_parts():
         )
 
 
-def test_deserialize_error_for_mismatched_protected_header():
+def test_deserialize_error_for_mismatched_protected_header_alg():
     with pytest.raises(ValueError):
         offchain.jws.deserialize(
-            b".".join([b"header", b"payload", b"sig"]),
+            b".".join(map(lambda t: base64.urlsafe_b64encode(t), [b'{"alg": "none"}', b"payload", b"sig"])),
```

Comment:
instead of lambda I think you can just pass a meth, i.e `base64.urlsafe_b64encode`